src/components/metadata/metadata.test.js

```diff
@@ -20,6 +22,8 @@ describe('MetaData', () => {
       afterLayoutActions: [
         // Click the expected node
         () => toggleNodeClicked(props.nodeId),
+        //simulating loadNodeData in node.js
+        () => addNodeMetadata({ id: props.nodeId, data: props.mockMetadata }),
```

Comment:
It looks like this is added to simulate the `input` and `output` data field that is now returned from the endpoint -  rather than adding this to the mount function that affects all tests, it is simplier to simply adopt the same set up as the test for `code toggle`, where the `afterLayoutAction` method of simulating the fetching of node data is done only on the mock metadata rather than on the actual mount function itself.), hence reducing the need to add the `mockMetadata` for tests that does not need the mock metadata. 

Introducing the `afterLayoutAction` here would effectively introduce a lot of redundant refactor of this test.